bedrock/firefox/tests/test_base.py

```diff
@@ -780,23 +780,23 @@ def test_fxdev_firstrun_57_0(self, render_mock):
     def test_fx_firstrun_57_0(self, render_mock):
         """Should use 57 quantum firstrun template"""
         req = self.rf.get("/en-US/firefox/firstrun/")
-        self.view(req, version="57.0")
-        template = render_mock.call_args[0][1]
-        assert template == ["firefox/firstrun/firstrun.html"]
+        resp = self.view(req, version="57.0")
+        assert resp.status_code == 301
+        assert resp["location"].endswith("/firefox/new/?reason=outdated")
 
-    # test redirect to /firefox/new/ for legacy /firstrun URLs - Bug 1343823
+    # test direct to /firefox/new/?reason=outdated for legacy /firstrun URLs with new notification bar to update browser
 
     @override_settings(DEV=True)
     def test_fx_firstrun_legacy_redirect(self, render_mock):
         req = self.rf.get("/firefox/firstrun/")
         req.locale = "en-US"
         resp = self.view(req, version="39.0")
         assert resp.status_code == 301
-        assert resp["location"].endswith("/firefox/new/")
+        assert resp["location"].endswith("/firefox/new/?reason=outdated")
 
     def test_fx_firstrun_dev_edition_legacy_redirect(self, render_mock):
         req = self.rf.get("/firefox/firstrun/")
         req.locale = "en-US"
         resp = self.view(req, version="39.0a2")
         assert resp.status_code == 301
-        assert resp["location"].endswith("/firefox/new/")
+        assert resp["location"].endswith("/firefox/developer/")
```

Comment:
Question: We can probably delete these tests too now since they are already covered by the tests above?